README.md

```diff
@@ -1 +1,161 @@
+<a name="readme-top"></a>
 
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [Budget App ](#budget-app-)
+  - [video demo 🎥](#video-demo-)
+  - [Render Deploy 🚀](#render-deploy-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Budget App <a name="about-project"></a>
+
+> This mobile web app allows users to create categories and add transactions to each category. The app will show the total amount spent on each category. The app is built with Ruby on Rails. The app is deployed on Render.
+
+## video demo 🎥
+
+[video demo here](https://www.loom.com/share/3cd7d67c0f9640f790fbe998a70a4130)
```

Comment:
- [ ] Great job on adding the video presentation 👍, however according to the requirements, while demonstrating the project you have to verbally explain the project in such a way that is clearly audible in the video presentation 🔉 . 

- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.

- The video contains a demo of the project, highlighting all the project features explained by you in professional english.

- The video contains a highlight about a piece of the code you find interesting or is proud of making explained by you in professional english.

To summarize please record a new video with you explaining the project in clear english... Cheers 👍